docs/csharp/tour-of-csharp/types-and-variables.md

```diff
@@ -72,35 +72,35 @@ This summarizes C#’s numeric types.
     - `float`  : 32 bits, range from 1.5 × 10^−45 - 3.4 × 10^38,    7-digit precision
 	- `double` : 64 bits, range from 5.0 × 10^−324 - 1.7 × 10^308, 15-digit precision
 * Decimal
-    - `decimal : 128 bits, range is at least –7.9 × 10^−28 -  7.9 × 10^28, with at least 28-digit precision
+    - `decimal` : 128 bits, range is at least –7.9 × 10^−28 -  7.9 × 10^28, with at least 28-digit precision
```

Comment:
Good catch of the missing backtick here. The use of `<sup>` could improve readability. @BillWagner, @rpetrusha any thoughts on this?